### ISSUE TITLE
acoustic camera

### ISSUE THREAD

**Issue by taospartan** [introlab/odas#30]
Has odas ever been used as part of an acoustic camera?

**Comment by FrancoisGrondin** [comment]
We did use odas in the past to add a tag on the image to display where the active sources were. I think what you want is different. If I understand your question, you would like to plot some sort of "heat layer" that contains the acoustic energy for all direction of arrivals, and to project it over the image. Is this what you would like to achieve?

**Comment by taospartan** [comment]
Yes that is the idea, a heat map that allows the user to see the origin of the sound

**Comment by FrancoisGrondin** [comment]
In this case, no, unfortunately, this feature is not implemented yet. If you feel this can be useful, this is something I'll implement.

**Comment by taospartan** [comment]
I think this feature would greatly help many projects, as acoustic cameras can be outside the budget of most.

**Comment by FrancoisGrondin** [comment]
Sure, I'll try to add this feature during my spare time in the coming days then.

**Comment by taospartan** [comment]
You are very kind!😀

**Comment by ketronium** [comment]
@taospartan did you have any luck building your acoustic camera?
